charts/memgraph-high-availability/templates/coordinators.yaml

```diff
@@ -9,28 +9,85 @@ spec:
   selector:
     matchLabels:
       app: memgraph-coordinator-{{ $coordinator.id }}
+      role: coordinator
   template:
     metadata:
       labels:
         app: memgraph-coordinator-{{ $coordinator.id }}
+        role: coordinator
```

Comment:
instance-type and role will be merged into one because they mean the same thing. @as51340 will remove it